src/gossip/service.zig

```diff
@@ -426,12 +434,15 @@ pub const GossipService = struct {
             exit_condition.ordered.exit_index += 1;
         }
 
-        try self.service_manager.spawn("[gossip] sendSocket", socket_utils.sendSocket, .{
+        // SocketPipe overrides `packet_outgoing_channel.send_hook`
```

Comment:
Is this accurate? I don't see this happening anywhere inside SocketPipe.

Reply:
Not atm, leftover comment from when SocketPipe setup the ResetEvent SendHook itself (now its been moved into the channel).

Reply:
https://github.com/Syndica/sig/pull/486/commits/767607c3c46beb88cf4533d9bbabba71b9b3e634